scripts/populate_conferences.py

```diff
@@ -92,11 +111,14 @@ def main():
             'conferences',
             'bitss.jpg',
         ),
+        'location': 'Berkeley, CA',
+        'start_date': 'Dec 11 2014',
+        'end_date': 'Dec 12 2014',
         'active': False,
         'admins': [
             'gkroll@berkeley.edu',
             'awais@berkeley.edu',
-        ],
+         ],
```

Comment:
I don't think it was intended, though I did come across other spacing issues. This was also reviewed by Chad before I got it.

Space removed.